bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieRequestProcessor.java

```diff
@@ -289,38 +293,33 @@ private void processStartTLSRequestV3(final BookkeeperProtocol.Request r, final
             c.writeAndFlush(response.build());
         } else {
             // there is no need to execute in a different thread as this operation is light
-            try {
-                SslHandler sslHandler = shFactory.getBookieHandler();
-                c.pipeline().addFirst("ssl", sslHandler);
+            SslHandler sslHandler = shFactory.newSslHandler();
+            c.pipeline().addFirst("ssl", sslHandler);
 
-                response.setStatus(BookkeeperProtocol.StatusCode.EOK);
-                BookkeeperProtocol.StartTLSResponse.Builder builder = BookkeeperProtocol.StartTLSResponse.newBuilder();
-                response.setStartTLSResponse(builder.build());
-                sslHandler.handshakeFuture().addListener(new GenericFutureListener<Future<Channel>>() {
-                    @Override
-                    public void operationComplete(Future<Channel> future) throws Exception {
-                        // notify the AuthPlugin the completion of the handshake, even in case of failure
-                        AuthHandler.ServerSideHandler authHandler = c.pipeline()
-                                .get(AuthHandler.ServerSideHandler.class);
-                        authHandler.authProvider.onProtocolUpgrade();
-                        if (future.isSuccess()) {
-                            LOG.info("Session is protected by: {}", sslHandler.engine().getSession().getCipherSuite());
-                        } else {
-                            LOG.info("SSL Handshake failure: {}", future.cause());
+            response.setStatus(BookkeeperProtocol.StatusCode.EOK);
+            BookkeeperProtocol.StartTLSResponse.Builder builder = BookkeeperProtocol.StartTLSResponse.newBuilder();
+            response.setStartTLSResponse(builder.build());
+            sslHandler.handshakeFuture().addListener(new GenericFutureListener<Future<Channel>>() {
+                @Override
+                public void operationComplete(Future<Channel> future) throws Exception {
+                    // notify the AuthPlugin the completion of the handshake, even in case of failure
+                    AuthHandler.ServerSideHandler authHandler = c.pipeline()
+                            .get(AuthHandler.ServerSideHandler.class);
+                    authHandler.authProvider.onProtocolUpgrade();
+                    if (future.isSuccess()) {
+                        LOG.info("Session is protected by: {}", sslHandler.engine().getSession().getCipherSuite());
```

Comment:
maybe this could be 'debug' and not 'info' (it was a debug for the auth plugin integration)